[2024-05-20T16:00:34.378+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_data_attendance_test_naja.delete_att_data_into_db scheduled__2024-05-19T01:00:00+00:00 [queued]>
[2024-05-20T16:00:34.408+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_data_attendance_test_naja.delete_att_data_into_db scheduled__2024-05-19T01:00:00+00:00 [queued]>
[2024-05-20T16:00:34.413+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-05-20T16:00:34.457+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): delete_att_data_into_db> on 2024-05-19 01:00:00+00:00
[2024-05-20T16:00:34.480+0000] {standard_task_runner.py:60} INFO - Started process 17504 to run task
[2024-05-20T16:00:34.505+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'api_data_attendance_test_naja', 'delete_att_data_into_db', 'scheduled__2024-05-19T01:00:00+00:00', '--job-id', '967', '--raw', '--subdir', 'DAGS_FOLDER/api_data_attendance.py', '--cfg-path', '/tmp/tmphhtd8837']
[2024-05-20T16:00:34.539+0000] {standard_task_runner.py:88} INFO - Job 967: Subtask delete_att_data_into_db
[2024-05-20T16:00:34.863+0000] {task_command.py:423} INFO - Running <TaskInstance: api_data_attendance_test_naja.delete_att_data_into_db scheduled__2024-05-19T01:00:00+00:00 [running]> on host e3e2dc6bc2be
[2024-05-20T16:00:35.425+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_data_attendance_test_naja' AIRFLOW_CTX_TASK_ID='delete_att_data_into_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-19T01:00:00+00:00'
[2024-05-20T16:00:35.479+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T16:00:35.536+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T16:00:35.600+0000] {sql.py:450} INFO - Running statement: 
        DELETE FROM attendance;
        , parameters: None
[2024-05-20T16:00:35.622+0000] {sql.py:459} INFO - Rows affected: 274
[2024-05-20T16:00:35.632+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-20T16:00:35.733+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=api_data_attendance_test_naja, task_id=delete_att_data_into_db, execution_date=20240519T010000, start_date=20240520T160034, end_date=20240520T160035
[2024-05-20T16:00:36.135+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-20T16:00:36.562+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
